source/index.html

```diff
@@ -1,39 +1,282 @@
-    <script>
-        document.getElementById('main').style.backgroundImage = [
-            'radial-gradient(',
-                'circle farthest-side at center,',
-                'hsl(' + Math.floor(Math.random() * 360) + ', 50%, 50%) 0%,',
-                'hsl(' + Math.floor(Math.random() * 360) + ', 50%, 50%) 100%',
-            ')'
-        ].join('');
+    <main>
+        <header>
+            <div>
+                <nav>
+                    <a href="#about-me">About Me</a>
+                    <a href="#grooming">Grooming</a>
+                    <a href="#daycare">Daycare</a>
+                    <a href="#boarding">Boarding</a>
+                    <a href="#clients">Clients</a>
+                </nav>
+            </div>
+            <div class="header">
+                <img src="./assets/pet-parlor.png">
+                <h1 class="title">POET'S PET PARLOR</h1>
+                <table class="groom-time">
+                    <th colspan="4">Grooming</th>
+                    <tr>
+                        <td>M</td>
+                        <td>8-5</td>
+                        <td>Th</td>
+                        <td>8-8<span>*</span></td>
+                    </tr>
+                    <tr>
+                        <td>T</td>
+                        <td>8-5</td>
+                        <td>F</td>
+                        <td>8-5<span>*</span></td>
+                    </tr>
+                    <tr>
+                        <td>W</td>
+                        <td>8-8</td>
+                        <td>S</td>
+                        <td>8-2*</td>
+                    </tr>
+                    <td colspan="4">*by appointment</td>
+                </table>
+                <table class="daycare-time">
+                    <th colspan="4">Daycare</th>
+                    <tr>
+                        <td>M</td>
+                        <td>7-7</td>
+                        <td>Th</td>
+                        <td>7-7<span>*</span></td>
+                    </tr>
+                    <tr>
+                        <td>T</td>
+                        <td>7-7</td>
+                        <td>F</td>
+                        <td>7-7<span>*</span></td>
+                    </tr>
+                    <tr>
+                        <td>W</td>
+                        <td>7-7</td>
+                        <td>S</td>
+                        <td>7-7*</td>
+                    </tr>
+                    <td colspan="4">*by appointment</td>
+                </table>
+                <div class="contact-us">
+                    <a href="https://www.google.com/maps/place/Poets+Pet+Parlor/@46.5646401,-87.415697,17z/data=!3m1!4b1!4m5!3m4!1s0x4d51df4d52a6b0cd:0xfc299af6713a58b7!8m2!3d46.5646401!4d-87.4135083"><img src="./assets/location.svg"></a>
+                    <a href="tel:(906)228-2708"><img src="./assets/phone.svg"></a>
+                    <a href="mailto:poetspetparlor@gmail.com"><img src="./assets/email.svg"></a>
```

Comment:
All the `img` tags here should have `alt=""` (empty `alt` attributes are a signal to screen readers and web crawlers like Google that these are icons and not "image-content").